src/TestableIO.System.IO.Abstractions.Wrappers/FileSystemWatcherBase.cs

```diff
@@ -77,7 +83,7 @@ public void Dispose()
                 return null;
             }
 
-            return new FileSystemWatcherWrapper(watcher);
+            return new FileSystemWatcherWrapper(null, watcher);
```

Comment:
Can we do the same like for the other wrappers, e.g. https://github.com/TestableIO/System.IO.Abstractions/blob/8a24fd27db06503654d0a9856818e797e4898a7f/src/TestableIO.System.IO.Abstractions.Wrappers/FileInfoBase.cs#L92?

```suggestion
            return new FileSystemWatcherWrapper(new FileSystem), watcher);
```